internal/tiltfile/tiltfile_test.go

```diff
@@ -2764,6 +2764,11 @@ func (f *fixture) assertNextManifest(name string, opts ...interface{}) model.Man
 						}
 						matcher.fb.checkMatchers(f, m, *dbInfo.FastBuild)
 					}
+				case model.LiveUpdate:
+					lu := image.MaybeLiveUpdateInfo()
+					if assert.NotNil(f.t, lu) {
+						assert.Equal(f.t, matcher, *lu)
```

Comment:
![this is fine](https://user-images.githubusercontent.com/6332648/55255237-7a94f100-5230-11e9-9fc7-ebd2eec73e2d.jpg)